packages/vector_graphics_compiler/lib/src/util.dart

```diff
@@ -20,3 +20,13 @@ bool listEquals<T>(List<T>? a, List<T>? b) {
   }
   return true;
 }
+
+/// Linearly interpolates between two doubles by factor t.
+@pragma('vm:prefer-inline')
```

Comment:
nit: we probably don't care about inline pragma on this lerpDouble in the compiler

Reply:
I CARE JONAH, I CARE